examples/golang/build.go

```diff
@@ -124,6 +135,10 @@ func Shutdown() {
 	root.Shutdown()
 }
 
+func buildContainer() string {
+	return fmt.Sprint("build:", version)
```

Comment:
nit: it was a problem in the original, but might be better to call this magnet-example-builder or something that looking at the docker image list is less likely to be confused by.